//
// Created by anyone on 15/04/21.
//

#ifndef SRC_PARAMETERS_H
#define SRC_PARAMETERS_H

namespace CARES{
    namespace TraditionalStereo{
        const std::string LEFT_IMAGE_S  = "left_image";
        const std::string RIGHT_IMAGE_S = "right_image";
        const std::string STEREO_INFO_S = "stereo_info";

        const std::string POINT_CLOUD_S = "point_cloud";
        const std::string DEPTH_IMAGE_S = "depth_image";

        const std::string SCALE_D = "scale";
    }
}

#endif //SRC_PARAMETERS_H
